cmd/esc/cli/testdata/env-set-file.yaml

```
@@ -0,0 +1,80 @@
+run: |
+  echo "hello" | esc env set default/test foo -f=-
+  esc env get default/test
+  echo "hello world" >test.file
+  esc env set default/test foo -f=test.file
+  esc env get default/test
+  esc env get default/test --value string
+  esc env set default/test foo -f=test.file --string
+  esc env get default/test
+  esc env set default/test -f=test.file || echo -n ""
+  esc env set -f=test.file || echo -n ""
+environments:
+  test-user/default/test:
+    values:
+      foo: bar
```

we could use a test for invalid UTF-8 in both stdin and a file on disk. you _should_ be able to use the `process` property to add files that are accessible to the commands in `run`:

```suggestion
  echo "hello" | esc env set default/test foo -f=-
  esc env get default/test
  echo "hello world" >test.file
  esc env set default/test foo -f=test.file
  esc env get default/test
  esc env get default/test --value string
  esc env set default/test foo -f=test.file --string
  esc env get default/test
  esc env set default/test -f=test.file || echo -n ""
  esc env set -f=test.file || echo -n ""
  esc env set default/test -f=binary foo || echo -n ""
  esc env set -f=- default/test foo <binary || echo -n ""
environments:
  test-user/default/test:
    values:
      foo: bar
process:
  fs:
    binary: !!binary sFOZJGpvmbYHwHRLnCYSnQ==
```